src/averageAges.js

```diff
@@ -55,7 +81,21 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const filteredTab = (onlyWithSon)
```

Comment:
Find better name so your code is easier to read. Now it is hard to guess what is in filterTab const

Reply:
```suggestion
  const children= (onlyWithSon)
```